code/modules/admin/chat_commands.dm

```diff
@@ -0,0 +1,73 @@
+GLOBAL_LIST_EMPTY(chat_round_end_notifiees)
+#define IRC_STATUS_THROTTLE 5
+
+/datum/server_tools_command/ircstatus
+	name = "status"
+	help_text = "Gets the admincount, playercount, gamemode, and true game mode of the server"
+	admin_only = TRUE
+	var/static/last_irc_status = 0
+
+/datum/server_tools_command/ircstatus/Run(sender, params)
+	var/rtod = REALTIMEOFDAY
+	if(rtod - last_irc_status < IRC_STATUS_THROTTLE)
+		return
+	last_irc_status = rtod
+	var/list/adm = get_admin_counts()
+	var/list/allmins = adm["total"]
+	var/status = "Admins: [allmins.len] (Active: [english_list(adm["present"])] AFK: [english_list(adm["afk"])] Stealth: [english_list(adm["stealth"])] Skipped: [english_list(adm["noflags"])]). "
+	status += "Players: [GLOB.clients.len] (Active: [get_active_player_count(0,1,0)]). Mode: [SSticker.mode ? SSticker.mode.name : "Not started"]."
+	return status
+
+/datum/server_tools_command/irccheck
+	name = "check"
+	help_text = "Gets the playercount, gamemode, and address of the server"
+	admin_only = TRUE
+	var/static/last_irc_check = 0
+
+/datum/server_tools_command/irccheck/Run(sender, params)
+	var/rtod = REALTIMEOFDAY
+	if(rtod - last_irc_check < IRC_STATUS_THROTTLE)
+		return
+	last_irc_check = rtod
+	return "[GLOB.round_id ? "Round #[GLOB.round_id]: " : ""][GLOB.clients.len] players on [SSmapping.config.map_name], Mode: [GLOB.master_mode]; Round [SSticker.HasRoundStarted() ? (SSticker.IsRoundInProgress() ? "Active" : "Finishing") : "Starting"] -- [config.server ? config.server : "[world.internet_address]:[world.port]"]" 
+
+/datum/server_tools_command/ahelp
+	name = "ahelp"
+	help_text = "<ckey> <message|ticket <close|resolve|icissue|reject|reopen <ticket #>|list>>"
+	required_parameters = 2
+	admin_only = TRUE
+
+/datum/server_tools_command/ahelp/Run(sender, params)
+	var/list/all_params = splittext(params, " ")
+	var/target = all_params[1]
+	all_params.Cut(1, 2)
+	return IrcPm(target, all_params.Join(" "), sender)
+
+/datum/server_tools_command/namecheck
+	name = "namecheck"
+	help_text = "Returns info on the specified target"
+	required_parameters = 1
+	admin_only = TRUE
+
+/datum/server_tools_command/namecheck/Run(sender, params)
+	log_admin("IRC Name Check: [sender] on [params]")
+	message_admins("IRC name checking on [params] from [sender]")
+	return keywords_lookup(params, 1)
+
+/datum/server_tools_command/adminwho
+	name = "adminwho"
+	help_text = "Lists administrators currently on the server"
+	admin_only = TRUE
+
+/datum/server_tools_command/adminwho/Run(sender, params)
+	return ircadminwho()
+
+/datum/server_tools_command/notify
+	name = "notify"
+	help_text = "Pings the invoker when the running round ends"

```

Comment:
@Shadowlight213 you wanted this?

Reply:
Nbot already has this. I just need to change the trigger wording a bit to fix it.
Mso wanted something different where the notify would work even if the relay/bots went down, which I know how to do but haven't gotten around to due to various reasons. Adding this to the server tools leaves it at the mercy of the relay working, which doesn't really do what mso wanted.

Reply:
Relays never break with the new version

Reply:
Oh boy. Screenshotting this for when they do.
Anyways still doesn't do what mso wanted, and is kinda redundant.